src/Compilers/CSharp/Portable/Symbols/Source/SourceModuleSymbol.cs

```diff
@@ -83,6 +83,8 @@ internal override Machine Machine
                         return Machine.ArmThumb2;
                     case Platform.X64:
                         return Machine.Amd64;
+                    case Platform.Arm64:
+                        return (Machine)0xAA64;//Machine.Arm64; https://github.com/dotnet/roslyn/issues/25185
```

Comment:
>0xAA64 [](start = 40, length = 6)

Can we add an internal constant somewhere in the compiler so we don't have to keep writing this magic number?

Reply:
No, this work is intentionally avoiding any cross module changes to public or even internal surface area to avoid potential issues with making this change affect any of the Roslyn api surface, etc.